resources/css/bem/season-stats-popup.less

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+.season-stats-popup {
+  @_gap: 10px;
+
+  background: hsl(var(--hsl-b4));
+  border-radius: @border-radius--extra;
+  font-size: @font-size--normal;
+
+  &__content {
+    padding: 15px;
+    gap: @_gap;
+    display: grid;
+
+    &--stats {
+      grid-template-columns: 1fr auto;
+    }
+
+    &--top {
+      justify-content: center;
+      background: hsl(var(--hsl-b5));
+      border-top-left-radius: inherit;
+      border-top-right-radius: inherit;
+    }
+  }
+
+  &__division {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-left: calc(@_gap * 2);
+    border-left: 2px solid hsl(var(--hsl-c1));
+  }
+
+  &__img {
+    height: 35px;
+    border-radius: @border-radius-small;
+  }
+
+  &__main {
```

Comment:
I find the alignment weird. It's centered but the separator in the middle make it look like the separator should be exactly on the center except it's probably not.

Maybe have the ranking on the left as small as possible with `auto auto 1fr` (with actual line div so a simple 15px gap can be used to match the padding), and centered image section.

Or maybe even move the rank to be another row (but with larger value size) along with total score

Reply:
not sure what's the advantage of using ``auto auto 1fr`` than just using flex, unless i am missing what you mean exactly?

Reply:
with flex you need to specify `flex: none` to each relevant nodes

Reply:
looking at it now, yeah i didn't really get what you mean exactly in the first place...

wondering if ``1fr auto 1fr`` wouldn't maybe be better so that the line just always stays in the center?

Reply:
I think the rank number area on the left looks way too empty compared to its right if the line is on exact center

Reply:
went with ``auto auto 1fr`` for now then